test.js

```diff
@@ -72,10 +59,16 @@ test.serial('pkg.bin as a string should work', t => {
 	});
 
 	t.is(process.title, 'browser-sync');
-	t.end();
 });
 
 test('single character flag casing should be preserved', t => {
 	t.ok(fn({argv: ['-F']}).flags.F);
-	t.end();
+});
+
+test('type inference', t => {
+	t.is(fn({argv: ['5']}).input[0], '5');
+	t.is(fn({argv: ['5']}, {string: ['_']}).input[0], '5');
+	t.is(fn({argv: ['5'], inferType: true}).input[0], 5);
+	t.is(fn({argv: ['5'], inferType: true}, {string: ['foo']}).input[0], 5);
+	t.is(fn({argv: ['5'], inferType: true}, {string: ['_', 'foo']}).input[0], 5);
```

Comment:
Is this correct? What if the user sets `inferType` to `true` but passes `_` in the string property? In my opinion this is correct as `inferType` has "higher priority" then the `string` property.